tests/Umbraco.Tests.AcceptanceTest/tests/DefaultConfig/BlockGridEditor/Datatype/BlockGridEditorDataTypeBlocks.spec.ts

```diff
@@ -0,0 +1,738 @@
+    await umbracoUi.clickDataElementByElementName(ConstantHelper.actions.delete);
+    await umbracoUi.clickElement(umbracoUi.getButtonByLabelKey(ConstantHelper.buttons.delete));
+
+    // Assert
+    // Checks if the block grid editor still exists
+    await umbracoUi.goToSection(ConstantHelper.sections.settings);
+    await umbracoUi.clickDataElementByElementName('tree-item-dataTypes', {button: "right"});
+    await umbracoUi.clickDataElementByElementName('action-refreshNode');
+    await expect(page.locator('[data-element="tree-item-dataTypes"] >> [data-element="tree-item-' + blockGridType + '"]')).not.toBeVisible();
+    await expect(await umbracoApi.dataTypes.exists(blockGridName)).toBe(false);
+  });
+
+  test('can move an element in a block grid editor to another group', async ({page, umbracoApi, umbracoUi}) => {
+    await umbracoApi.documentTypes.ensureNameNotExists(elementName);
+
+    const GroupMoveHere = 'MoveToHere';
+
+    const element = await umbracoApi.documentTypes.createDefaultElementType(elementName, elementAlias);
+
+    const blockGridType = new BlockGridDataTypeBuilder()
+      .withName(blockGridName)
+      .addBlockGroups()
+        .withName(GroupMoveHere)
+      .done()
+      .addBlock()
+        .withContentElementTypeKey(element['key'])
+        .withLabel('Did we move?')
+      .done()
+      .build();
+    await umbracoApi.dataTypes.save(blockGridType);
+
+    await umbracoUi.navigateToDataType(blockGridName);
+
+    // Drags the element from the default group to the 'MoveToHere' Group.
+    await page.locator('.umb-block-card-group').nth(0).locator('[data-content-element-type-key="' + element['key'] + '"]').dragTo(page.locator('[key="blockEditor_addBlockType"]').nth(1));
+    await umbracoUi.clickElement(umbracoUi.getButtonByLabelKey(ConstantHelper.buttons.save));
+
+    // Assert
+    await umbracoUi.isSuccessNotificationVisible();
+    // Checks if the elements was moved to the correct group
+    await expect(page.locator('.umb-block-card-group').nth(1).locator('[data-content-element-type-key="' + element['key'] + '"]')).toBeVisible();
+    // Checks if the element still contains the correct text
+    await page.locator('.umb-block-card-group').nth(1).locator('[data-content-element-type-key="' + element['key'] + '"]').click();
+    await expect(page.locator('input[name="label"]')).toHaveValue('Did we move?');
+    await umbracoUi.clickElement(umbracoUi.getButtonByLabelKey('general_close'));
+    // Checks if the datatype was created
+    await expect(await umbracoApi.dataTypes.exists(blockGridName)).toBe(true);
+    await umbracoUi.doesDataTypeExist(blockGridName);
+
+    // Clean
+    await umbracoApi.documentTypes.ensureNameNotExists(elementName);
+  });
+
+  test('can move an empty group in a block grid editor', async ({page, umbracoApi, umbracoUi}) => {
+    const GroupMove = 'MeMove';
+    const GroupNotMoving = 'MeStay';
```

Comment:
These names gave me a laugh. The text should probably just be the same name as the variables. So "GroupMove" or "GroupToMove"